Creation of Registration company Interface and Create views for Companies

SEVERO96 opened the pull request:
CRUD of the module of the branches

ManuelAcostaGuerrero committed:
Creation of Registration company Interface and Create views for Companies

I report the total GRUD of the module of the branches

YilKanda commented on checador/app/assets/stylesheets/companies.scss:
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the companies controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
If you don't occupy this file, delete it.

ManuelAcostaGuerrero replied:
Thanks, the changes were made, could you check

YilKanda commented on checador/app/helpers/companies_helper.rb:
```diff
@@ -0,0 +1,2 @@
+module CompaniesHelper
+end
```
If you don't occupy this file, delete it.

ManuelAcostaGuerrero replied:
Thanks, the changes were made, could you check

YilKanda commented on checador/spec/helpers/companies_helper_spec.rb:
```diff
@@ -1,5 +1,15 @@
 require 'rails_helper'
 
+# Specs in this file have access to a helper object that includes
+# the CompaniesHelper. For example:
+#
+# describe CompaniesHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
```
If you do not occupy this code, delete it.

ManuelAcostaGuerrero replied:
Thanks, the changes were made, could you check

YilKanda reviewed (commented):
As a suggestion, be more specific with your commit and your PR, you explain that you did the CRUD of the branches, but how does that change benefit the project? Why did it need to be implemented? This can provide a general idea of ​​what was done in your PR.

jclopezdev commented on checador/app/models/company.rb:
```diff
@@ -0,0 +1,2 @@
+class Company < ApplicationRecord
```
What do you think of adding some validations of the model here? I want to think that you don't need a Company without a `name`, and also add some specs for this model

ManuelAcostaGuerrero replied:
Hello, the validations are intended to be generated by having the modules fully integrated. For now we think we better leave it like this

jclopezdev commented on checador/db/migrate/20210723044029_create_companies.rb:
```diff
@@ -0,0 +1,10 @@
+  def change
+    create_table :companies do |t|
+      t.string :name
+      t.string :addres
```
You have a `typo` here, I think you mean `address`

ManuelAcostaGuerrero replied:
Thanks, the changes were made, could you check

ManuelAcostaGuerrero committed:
Deletion of .scss file that does not take care
The file path:
checador / app / assets / stylesheets / companies.scss

ManuelAcostaGuerrero committed:
Delete the file that is not needed in helpers

borrado:        app/helpers/companies_helper.rb

ManuelAcostaGuerrero committed:
Delete the file that is not needed in spec/helpers

borrado:        spec/helpers/companies_helper_spec.rb

ManuelAcostaGuerrero committed:
Merge branch 'master' into company

jclopezdev commented on checador/app/views/companies/index.html.erb:
```diff
@@ -1 +1,30 @@
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Addres</th>
```
You also need to update the `Addres` word here

ManuelAcostaGuerrero replied:
Ready Thank you very much we are attentive to any changes

jclopezdev commented on checador/app/controllers/companies_controller.rb:
```diff
@@ -1,41 +1,71 @@
+
+    # Only allow a list of trusted parameters through.
+    def company_params
+      params.require(:company).permit(:name, :addres)
```
And here...

ManuelAcostaGuerrero replied:
Ready Thank you very much we are attentive to any changes

jclopezdev commented on checador/app/views/companies/_form.html.erb:
```diff
@@ -0,0 +1,27 @@
+  </div>
+
+  <div class="field">
+    <%= form.label :addres %>
```
And also here...

xFerchoVJ replied:
Thank you, Juan Carlos!
We're going to change it!

SEVERO96 replied:
ok

ManuelAcostaGuerrero replied:
Ready Thank you very much we are attentive to any changes

jclopezdev reviewed (changes requested):
I can still see a lot of typos with the `address` word, please take a look to fix it in all the files.

ManuelAcostaGuerrero committed:
Fixed several bugs with the word address

 En la rama company
 Cambios a ser confirmados:
	modificado:     app/controllers/companies_controller.rb
	modificado:     app/views/companies/_company.json.jbuilder
	modificado:     app/views/companies/_form.html.erb
	modificado:     app/views/companies/index.html.erb

ManuelAcostaGuerrero committed:
Merge branch 'company' of https://github.com/BC-MAY-21-ROR/kata-06-reloj-checador-k06-t05 into company

YilKanda reviewed (approved):
LGTM!

xFerchoVJ committed:
Merge branch 'master' into company

xFerchoVJ reviewed (approved):
LGTM
